### ISSUE TITLE
New release w/compile fixes wanted

### ISSUE THREAD

**Issue by sjn** [ajgb/crypt-curve25519#13]
Hei!

Would you mind making a CPAN release of this package, with the fixes/pull requests applied? :-)

If you're too busy/can't bother, then please assign a co-maintainer to help you. I believe this can be done quite conveniently on [PAUSE](https://pause.cpan.org/). :grin: 

Thanks!

**Comment by sjn** [comment]
Hei!

Just checking in.

Would you consider listing this module as ADOPTME in CPAN/PAUSE? This could help find volunteers for the modules you don't have time to tend to.

Please check out http://blogs.perl.org/users/brian_d_foy/2013/02/mark-your-modules-as-adoptable-if-you-dont-want-them.html for some pointers on this topic. :-)

Thanks!

**Comment by willsheppard** [comment]
I would also like this, as I'm having to make manual fixes by following https://github.com/ajgb/crypt-curve25519/issues/9

I am assuming a new release will include that fix.

**Comment by 7olstoy** [comment]
Yeah, ansible with CPAN module also fly away without a new release

**Comment by kenneth-olwing** [comment]
+1, trying to build on RHEL 8.3 using Perl 5.30.0

**Comment by Capri205** [comment]
+1 will not compile on any platform I have tried - rhel8, centos8 or ubuntu 20.04. I put an issue ticket in, but probably best someone take this module over.

**Comment by willsheppard** [comment]
I emailed the maintainer ajgb's cpan address on 18th June, I've received no reply for over two weeks.

Does anyone have another contact address?

**Comment by sjn** [comment]
I sent mail to ajgb on February 13th, Feb. 22nd and April 16th, to his @ajgb.net address. Zero replies.
I think he's pretty much out of commission wrt. this project, and it's reasonable to contact the folks at modules@perl.org about having this module listed with the ADOPTME as a co-maintainer (this is how the PAUSE admins know that [the author has not been responsive, and the module needs a new custodian](https://neilb.org/2013/08/07/adoptme.html).)

**Comment by Capri205** [comment]
I found a patch someone had posted in another thread so you can compile it with that patch. These are my steps in my container build:

RUN wget http://www.cpan.org/authors/id/A/AJ/AJGB/Crypt-Curve25519-0.06.tar.gz && \
    tar -xvzf Crypt-Curve25519-0.06.tar.gz && \
    export LANG=C && \
    cd Crypt-Curve25519-0.06 && \
    wget https://www.cpan.org/authors/id/S/SR/SREZIC/patches/Crypt-Curve25519-0.06-PR10-ANOTHERLINK.patch && \
    git apply Crypt-Curve25519-0.06-PR10-ANOTHERLINK.patch && \
    perl Makefile.PL && \
    make && \
    make test && \
    make install

**Comment by sjn** [comment]
This module needs to be taken over. I've gathered some due diligence documentation for this: https://github.com/ajgb/crypt-curve25519/pull/10#issuecomment-950152224

If anyone is up for proceeding with this, please do so! :-)

**Comment by dk** [comment]
Hi all, Alex granted me a co-maint status and so there's v0.07 out - please check out of it works for you.
